types/lodash/ts3.1/common/object.d.ts

```diff
@@ -1097,6 +1097,16 @@ declare module "../index" {
          */
         get<TDefault>(path: number, defaultValue: TDefault): T | TDefault;
     }
+    interface Primitive<T> {
+        /**
+         * @see _.get
+         */
+        get(path: PropertyPath): any;
```

Comment:
I'm worried about losing type safety on intersection chain types - if any of the intersected types include `Primitive<undefined>` then the return type becomes `any`, which is unexpected. 

Maybe this should return something like:

```ts
T extends null | undefined ? undefined : any;
```

Reply:
I just deleted the Primitive.get since it's not needed to fix original bug after altering the Chain types.